proof-of-file-existence/pofe/readme.md

```diff
@@ -0,0 +1,31 @@
+# pofe
+
+**pofe** is a blockchain application built using Cosmos SDK and Tendermint and generated with [Starport](https://github.com/tendermint/starport).
```

Comment:
A few sentences on what pofe is would help. Some people who come to the tutorials may not know what this means and will be less enticed to go through it.